### ISSUE TITLE
Any plans for bevfusion TRT

### ISSUE THREAD

**Issue by lucianzhong** [DerryHub/BEVFormer_tensorrt#50]
Hi, 
Thanks for this great work.
Any plans for bevfusion TRT ?

**Comment by DerryHub** [comment]
Yes, I will try to implement quantization including but not limited to bevdet, bevdepth and bevfusion in the next few months.

**Comment by lucianzhong** [comment]
Looking forward to.  
:wink:
